automation/wdio.conf.js

```diff
@@ -88,16 +88,19 @@ exports.config = {
     ignoreUndefinedDefinitions: false,
   },
 
-  before: function (/* capabilities, specs */) {
+  before: function () {
     // Add commands to WebdriverIO
+    /* eslint-disable no-unused-expressions */
```

Comment:
chore (blocking): this eslint disable is doing nothing
```suggestion
```

Reply:
done